readme.md

```diff
@@ -132,6 +131,7 @@ Please check the <a href="https://github.com/tobiasbueschel/awesome-pokemon/blob
 - [Server Status](http://www.mmoserverstatus.com/pokemon_go) - Shows you if Pokémon GO is down.
 - [Egg Statistics](https://app.cmmcd.com/) - Pokémon GO Egg Reporter.
 - [Poke Assistant](https://pokeassistant.com/) - Collection of calculators for Pokémon GO.
+- [Calcy IV](https://play.google.com/store/apps/details?id=tesmath.calcy) - One-tap stats and level calculation on Pokemon GO (Android).
```

Comment:
Thanks for the PR @Letsan 🙏 Would you mind to correct the spelling of Pokémon here? Then I'll merge it in ASAP.

Sorry for the nitpick!

Reply:
I just fixed it ;)